src/Vgrep/Widget/Pager.hs

```diff
@@ -17,15 +17,15 @@ module Vgrep.Widget.Pager (
 
 import           Control.Lens         hiding ((:<), (:>))
 import           Data.Foldable
+import qualified Data.IntMap.Strict   as Map
 import           Data.Sequence        (Seq, (><))
 import qualified Data.Sequence        as Seq
-import qualified Data.Set             as Set
 import           Data.Text            (Text)
-import qualified Data.Text            as T
 import           Graphics.Vty.Image   hiding (resize)
 import           Graphics.Vty.Input
 import           Graphics.Vty.Prelude
```

Comment:
Actually, that's what I use. But I don't currently group qualified and unqualified imports separately.